de.html

```diff
@@ -0,0 +1,4 @@
+---
+layout: index
+lang: de
```

Comment:
Can this be modified like https://github.com/github/opensource.guide/commit/3dd15966d93ed26326a52fe008d7e570707a4f6e? Thanks!